source/_integrations/camera.mqtt.markdown

```diff
@@ -112,6 +112,11 @@ enabled_by_default:
   required: false
   type: boolean
   default: true
+encoding:
+  description: The encoding of the image payloads received. Set to `"b64"` to enable base64 decoding of image payload.
+  required: false
+  type: string
+  default: None
```

Comment:
We're documenting yaml, where the term is `null` not `None`.
@frenck Which term should we use?
```suggestion
  default: null
```